sched/scheduler/cluster_state.go

```diff
@@ -225,15 +224,17 @@ func (c *clusterState) update(updates []cluster.NodeUpdate) {
 			if ns, ok := c.suspendedNodes[update.Id]; ok {
 				if !ns.ready() {
 					// Adding a node that's already suspended as non-ready, leave it in that state until ready.
-					log.Infof("Suspended node re-added but still awaiting readiness check %v (%#v)", update.Id, ns)
+					log.Infof("Suspended node re-added but still awaiting readiness check %v (%s)", update.Id,
+						ns.node.String())
```

Comment:
I think you can just have ns.node here, the %s calls String() on it

Reply:
fixed and reverted to using ns